ui/components/multichain/multi-srp/import-srp/import-srp.test.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import {
+  createEvent,
+  fireEvent,
+  RenderResult,
+  waitFor,
+} from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { renderWithProvider } from '../../../../../test/jest/rendering';
+import mockState from '../../../../../test/data/mock-state.json';
+import * as actions from '../../../../store/actions';
+import { ImportSRP } from './import-srp';
+
+const mockClearClipboard = jest.fn();
+
+jest.mock('../../../../helpers/utils/util', () => ({
+  clearClipboard: () => mockClearClipboard(),
+}));
+
+const VALID_SECRET_RECOVERY_PHRASE =
+  'input turtle oil scorpion exile useless dry foster vessel knee area label';
+
+jest.mock('../../../../store/actions', () => ({
+  importMnemonicToVault: jest
+    .fn()
+    .mockReturnValue(jest.fn().mockResolvedValue(null)),
+  showAlert: jest.fn().mockReturnValue({ type: 'ALERT_OPEN' }),
+  hideAlert: jest.fn().mockReturnValue({ type: 'ALERT_CLOSE' }),
+}));
+
+const pasteSRPIntoFirstInput = async (render: RenderResult, srp: string) => {
+  const firstSRPWordDiv = render.getByTestId('import-srp__srp-word-0');
+  // This is safe because the input is always present in the word div.
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const firstSRPWordInput = firstSRPWordDiv.querySelector('input')!;
+
+  const pasteEvent = createEvent.paste(firstSRPWordInput, {
+    clipboardData: {
+      getData: () => srp,
+    },
+  });
+
+  fireEvent(firstSRPWordInput, pasteEvent);
+};
+
+describe('ImportSRP', () => {
+  const store = configureMockStore([thunk])(mockState);
+
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('enables the "Import wallet" button when a valid secret recovery phrase is entered', async () => {
+    const render = renderWithProvider(
+      <ImportSRP onActionComplete={jest.fn()} />,
+      store,
+    );
+    const { getByText } = render;
+
+    expect(getByText('Import wallet')).not.toBeEnabled();
+    await pasteSRPIntoFirstInput(render, VALID_SECRET_RECOVERY_PHRASE);
+
+    await waitFor(() => {
+      expect(getByText('Import wallet')).toBeEnabled();
+    });
```

Comment:
I think we should use this pattern in every other tests too. No need to repeat the `getByText` everytime:

```suggestion
    const importButton = getByText('Import wallet');
    
    expect(importButton).not.toBeEnabled();
    await pasteSRPIntoFirstInput(render, VALID_SECRET_RECOVERY_PHRASE);

    await waitFor(() => {
      expect(importButton)).toBeEnabled();
    });
```